### PR TITLE
Update runtime automatically on bottles start

### PR DESCRIPTION
**Pull request by cyberphantom52**
# Description
This ensures that the non flatpak version of bottles always uses the latest available runtime. The core prefrences section has also been removed to make the ui consistent between flatpak and non flatpak versions.

Fixes Nothing

## Type of change
- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. 
Provide instructions so we can reproduce.
- [X] Locally

### CODE REVIEW

File: bottles/backend/managers/manager.py [hunk 1]
```diff
@@ -410,35 +410,35 @@ def check_runners(self, install_latest: bool = True) -> bool:
...
-                except StopIteration:
-                    return False
-            return False
-
-        runtime = runtimes[0]  # runtimes cannot be more than one
-        manifest = os.path.join(Paths.runtimes, runtime, "manifest.yml")
+            logging.warning("No runtime found.")
+            install_latest = True
+        else:
+            runtime = runtimes[0]  # runtimes cannot be more than one
```
Comment (Kinsteen) [inline bottles/backend/managers/manager.py:427]: Are you sure? If users have 0.5 installed, then 0.6 will be installed. But then, both will be in the folder? There is nothing that deletes old version, unless I'm mistaken?

File: bottles/backend/managers/manager.py [hunk 2]
```diff
@@ -410,35 +410,35 @@ def check_runners(self, install_latest: bool = True) -> bool:
         self.runners_available = sorted(self.runners_available, reverse=True)
         return True
 
+    @RunAsync.run_async
     def check_runtimes(self, install_latest: bool = True) -> bool:
-        self.runtimes_available = []
+        RepoStatus.repo_wait_operation("components.fetching")
+        RepoStatus.repo_wait_operation("components.organizing")
         if "FLATPAK_ID" in os.environ:
```
Comment (Kinsteen) [inline bottles/backend/managers/manager.py:417]: This block should be removed for Flatpak, as we're using the components version

**Comment by cyberphantom52** [comment]
@Kinsteen i guess i should just close this PR and work on shipping EAC and BE as components?

**Comment by Kinsteen** [comment]
Yes, I think that would be good!

**Comment by cyberphantom52** [comment]
We decided that it was best to not ship runtime in the first place and rather switch to only shipping EAC and BattleEye runtimes.

**Pull request closed without merging**